src/clj/rems/api/services/catalogue.clj

```diff
@@ -17,7 +17,9 @@
                                        :items [id]})}))
 
 (defn create-catalogue-item! [{:keys [localizations] :as command}]
-  (let [id (:id (db/create-catalogue-item! (select-keys command [:form :resid :wfid :enabled :archived])))
+  ;; TODO make :organization unoptional?
+  (let [id (:id (db/create-catalogue-item! (merge {:organization ""}
```

Comment:
So, couldn't this "" come from the front-end instead of being filled up here, if the field is just left empty?

Reply:
ah I wanted to keep the API backwards-compatible, see TODO in the API file

Reply:
Ok right, never mind. It's fine.